shell/components/formatter/__tests__/PodsUsage.test.ts

```diff
@@ -2,37 +2,44 @@ import { mount } from '@vue/test-utils';
 import PodsUsage from '@shell/components/formatter/PodsUsage.vue';
 
 describe('component: PodsUsage', () => {
```

Comment:
I'm not even certain what that test would look like, how would that affect these tests?

Reply:
added this test since `mgmt` is just a getter to the cluster in the management store I went ahead and just removed the `mgmt` key from the row passed into the column.